client/sites/components/dotcom-style.scss

```diff
@@ -314,7 +314,7 @@
 
 .wpcom-site .hosting-dashboard-layout.sites-dashboard:not(.domains-overview) .site-preview-pane {
 	.hosting-dashboard-item-view__content {
-		> * {
+		> *:not(.site-logs) {
```

Comment:
Nit question: why added `not` instead of overriding `.site-logs`? Is it because order of loading?

Reply:
This rule is where the max-width is set for all the tabs, so I thought adding it here was more future-proof. I suppose adding a new rule to set max-width to none in the logs tab could work, but it has the disadvantage that isn't collocated with this rule, so future changes to this can easily miss it.